modApiServer/src/org/aion/api/server/rpc/ApiWeb3Aion.java

```diff
@@ -2461,10 +2399,10 @@ public RpcMsg ops_getTransactionReceiptListByBlockHash(Object _params) {
         // ok to getUnchecked() since the load() implementation does not throw checked exceptions
         AionBlock b;
         try {
-            b = blockCache.getUnchecked(new ByteArrayWrapper(blockHash));
-        } catch (CacheLoader.InvalidCacheLoadException e) {
+            b = blockCache.get(new ByteArrayWrapper(blockHash));
+        } catch (Exception e) {
```

Comment:
This catch got wider ... is there no equivalent more specific exception in Caffeine?

Reply:
Inside the catch block, there's a comment "Catch errors if send an incorrect tx hash" and then it returns RpcError.INVALID_REQUEST, which implies the user/caller provided input that's invalid.

I think in the original code, it's catching only cases where the given tx hash does not exist.  But the new catch block seems wider than that.  Would it be possible to catch only the exception that corresponds to the tx hash not existing (i think NullPointerException, when I took a look at LoadingCache.get()'s JavaDoc), and then another catch block for other exceptions, which returns some different error (internal error or something to that effect)?